docusaurus/docs/ui-extensions/create-an-extension-point.md

```diff
@@ -79,14 +79,48 @@ function AppMenuExtensionPoint() {
 }
 ```
 
-#### Why does the extension have `onClick` and `path`?
+### Why does the extension have `onClick` and `path`?
```

Comment:
Linked to the below comment, this might need to stay as h4 if an additional h3 is added for the above example

Reply:
Good point 👍

Reply:
Addressed in https://github.com/grafana/plugin-tools/pull/548/commits/a828ad4227e129281442655b3296a389642e6581.